semgrep-core/tests/OTHER/rules/taint_spread_record.js

```diff
@@ -0,0 +1,2 @@
+//ruleid: test
+sink({...source})
```

Comment:
What would `{...source}` be, some kind of conversion from one record type to another record type where the field names are shared?